package.json

```diff
@@ -45,8 +45,6 @@
     "LICENSE"
   ],
   "dependencies": {
-    "@types/quill": "1.3.10",
```

Comment:
As we fully bundle Quill inside ReactQuill, the `@types/quill` should also remain a regular dependency. Did you experience any issues with `@types/quill` specifically?